### PR TITLE
Suppress warning when running tests in Edge and update docs

### PR DESCRIPTION
**Pull request by mgrybyk**
## Proposed changes

- suppress `Warning: unknown product name edge. Falling back to chrome.` warning when running tests in Edge
- update devtools docs

## Types of changes

- [x] Polish
- [x] Documentation update

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have added proper type definitions for new commands (if appropriate)

### CODE REVIEW

File: packages/devtools/src/launcher.js [hunk 1]
```diff
@@ -108,7 +108,8 @@ async function launchChrome (capabilities) {
     return browser
 }
 
-function launchBrowser (capabilities, product) {
+function launchBrowser (capabilities, browserType) {
+    const product = browserType === BROWSER_TYPE.firefox ? BROWSER_TYPE.firefox : BROWSER_TYPE.chrome
```
Comment (mgrybyk) [inline packages/devtools/src/launcher.js:112]: puppeteer only supports firefox and chrome.

Comment (christian-bromann) [reply packages/devtools/src/launcher.js:112]: Chromium Edge as well

File: packages/devtools/src/constants.js [hunk 2]
```diff
@@ -125,7 +125,6 @@ export const BROWSER_ERROR_MESSAGES = {
 export const VENDOR_PREFIX = {
     chrome: 'goog:chromeOptions',
     firefox: 'moz:firefoxOptions',
-    edge: 'ms:edgeOptions'
```
Comment (christian-bromann) [inline packages/devtools/src/constants.js:128]: Chromium Edge is supported too!

File: packages/devtools/src/launcher.js (hunk 1 above)
Comment (mgrybyk) [reply packages/devtools/src/launcher.js:112]: Sorry for not being clear, `product` option support only `chrome` and `firefox`.

File: packages/devtools/src/constants.js (hunk 2 above)
Comment (mgrybyk) [reply packages/devtools/src/constants.js:128]: let's talk about it in one place https://github.com/webdriverio/webdriverio/pull/5943#discussion_r500648714

File: packages/devtools/src/launcher.js (hunk 1 above)
Comment (mgrybyk) [reply packages/devtools/src/launcher.js:112]: https://github.com/puppeteer/puppeteer/blob/950ae334ca9c4c37b5a34dc61596a89596fd2bc6/src/node/Launcher.ts#L803

Comment (mgrybyk) [reply packages/devtools/src/launcher.js:112]: I'm able to run tests in both headless and headful modes in the Edge browser, of course.
https://github.com/mgrybyk/webdriverio-devtools/blob/master/config/wdio.EDGE.conf.ts#L12

File: packages/devtools/src/constants.js (hunk 2 above)
Comment (christian-bromann) [reply packages/devtools/src/constants.js:128]: We still need this to properly generated [client capabilities](https://github.com/webdriverio/webdriverio/blob/master/packages/devtools/src/index.js#L40-L59)

Comment (mgrybyk) [reply packages/devtools/src/constants.js:128]: Assuming that user won't ever use "ms:edgeOptions" with devtools why would we need it?
Was not able to figure out the use case

Comment (mgrybyk) [reply packages/devtools/src/constants.js]: @christian-bromann reverted this one, however still don't understand why we need it.

**Review by christian-bromann — APPROVED** [review]
👍

**Pull request merged**